setup.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+ADAFRUIT_DIR=www/Adafruit_Python_GPIO
+
+if [ "${EUID}" -ne 0 ]; then
+	echo "Error: This script must be run as root."
+	exit 1
+fi
+
+if [ ! -e ${ADAFRUIT_DIR} ]; then
+	echo "Error: Adafruit directory not found."
+	exit 2
+fi
+
+apt-get install -y python python-dev python-pip python-smbus lighttpd
```

Comment:
I think including a default one would be better. This script has no idea what version of lighttpd will be installed, so trying to edit the default configuration file that might be slightly different between versions doesn't seem practical.